devices/gr716-multi/Makefile

```diff
@@ -0,0 +1,13 @@
+#
+# Phoenix-RTOS
+#
+# GR716 multi tests
+#
+# Copyright 2023 Phoenix Systems
+#
+
+NAME := test-gr716-multi
+LOCAL_SRCS := gr716-multi.c
+LIBS := unity
+
+include $(binary.mk)
```

Comment:
LGTM, I would only change source file name from gr716-multi.c to test-gr716-mulit and use add_test macro:
```
$(eval $(call add_test, test-gr716-multi, unity))
```